.github/workflows/dev.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
       - name: "Build, Tag, and push the Docker image"
         env:
-          IMAGE_NAME: ghcr.io/manimatter/decluttarr
```

Comment:
Sorry for the stupid question but, when you merged in ReadMe-Cleanup-&-ARM-Support, did you revert Fxsch's image name changes? 

@ManiMatter

Reply:
Not a stupid question at all. Yes I reverted intentionally because a error ‚unknown manifest‘ was reported that I was not able to solve

@craggles17  wanted to have a look into this, please see
https://github.com/ManiMatter/decluttarr/pull/20#issuecomment-1995926827

Reply:
Nvm, not the issue